content/pages/meetup/2023/1108-nycu.rst

```diff
@@ -17,15 +17,25 @@ Date
 Agenda
 --------
 
-* 18:30-19:30 Free discussion
-* 19:30-20:30 Discussion: Sciwork activity process automation
-* 20:30-21:30 Meeting: prepare November scisprint
+* 18:30-19:00 Free discussion
+* 19:00-20:30 Review nsd term project
```

Comment:
Could you please make the two sessions, project review and sciwork discussion, happen in parallel?  You do not need me for the sciwork automation discussion.

Reply:
Sure. I will modify it.

Reply:
I've modified it.
https://github.com/sciwork/swportal/pull/290